hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/translators/SingleTransactionBlockItems.java

```diff
@@ -34,7 +33,7 @@
  * @param output the output (if any) of processing the user transaction
  */
 public record SingleTransactionBlockItems(
-        @NonNull Transaction txn, @NonNull TransactionResult result, @Nullable TransactionOutput output) {
+        @Nullable Transaction txn, @Nullable TransactionResult result, @Nullable TransactionOutput output) {
```

Comment:
All of those nullables would be for the first pass only and in the future we'll need to revert to requiring non-null, am I right? If yes, then adding a TODO here as well might provide useful.

Reply:
I'm not sure. The sample tar file we're given has a bunch of blocks without these values. I'm not sure if these blocks are valid, or if they're just test objects until we generate more correct blocks